### ISSUE TITLE
Bug - sev-2 - ENG: Update buttons and links on Secure Messaging screens

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#7801]
The following list are the updates that need to be made to buttons and links on the secure messaging screens within the app.
_This list has been pulled in from the [full list of audit results](https://docs.google.com/spreadsheets/d/1uOF4dn3yptsp7VnxwG7owy2p0GDHjmBq1oBR8MIVvWg/edit?usp=sharing)._
- [ ] **More**
     - **Location:** Health / Messages / Drafts / Edit draft
     - **Current component type:** Button
     - **Current style:** N/A
     - **Icon:** Ellipsis
     - **What happens?** Shows popup menu with Save / Delete options
     - **Recommended change:** Should be a link not a button. Also currently says "menu, button". This should say "more, link" or "more, menu, link".

**Comment by brea11y** [comment]
Ticket has been reviewed and is still valid.

**Comment by DJUltraTom** [comment]
Verified this now says "more link" using ipad mini.
@brea11y is my sign off good enough for this or do you want to give it a once over?

**Comment by brea11y** [comment]
@DJUltraTom – Your sign off is great! We should be all set!

**Comment by timwright12** [comment]
@DJUltraTom can you approve in the PR for this so it can clear CI?

**Comment by DJUltraTom** [comment]
QA approves this change (and the PR)

**Comment by DJUltraTom** [comment]
verified this is working as expected in the QA overnight build
Closing as deployed
